add data from Philips2017

ivelsko opened the pull request:
# Pull Request

This PR is for a

- [x] [New Publication(s)](#new-publication)
- [ ] [Correction](#correction)

For the follow list (s)

- [x] Ancient Microbiome
- [ ] sedaDNA
- [ ] Ancient Pathogen

## New Publication

### Publication Information

<!-- Thank you for contributing to AncientMetagenomeDir Please fill in the information below-->
<!-- Please @ancientmetagenomedir-coreteam if you have any questions-->

This pull request is to add samples from the following publication(s):

Phillips  2017 to close #90 

<!-- Replace this comment with citation-->

### Checklist

- [x] Publication is published (preprints currently not accepted)?
- [x] Checked the publication is not already in the database?
- [x] Checked samples in this publication are not previously published _data_ (newly re-sequenced metagenomes are OK!)?
- [x] Samples are shotgun metagenomes?
- [x] Checked the list follows conventions as described in the corresponding sample type's README file (e.g. using ERS/SRS accession codes for ENA/SRA)?
- [ ] (If applicable) Updated the JSON files under `/assets/enums` with new categories (e.g. material or archive)?
- [x] Does your PR pass validation checks?
- [ ] Changelog is updated?

<details>
  <summary>If you do not know how to check errors in failed validation checks, expand here</summary>
  
   1. Press 'details' next to the failed check.
   2. Expand the `test ancient <list>` line with the red X next to it.
   3. Scroll to the bottom of the log, and look for a `DatasetValidationError` (usually the last line).
   4. Read the error, and fix accordingly. Check the README for a given list for more guidance. If in doubt, ask!

</details>

## Correction

This PR is for

- [x] Ancient Microbiome
- [ ] sedaDNA
- [ ] Ancient Pathogen

### Reference

This pull request is to correct samples from the following publication(s):

<!-- Replace this with the publication being corrected -->

### Description

The issue is: 

<!-- Replace this with a description and justification of the correction -->

### Checklist

- [x] Checked the corrected entries follow conventions as described in the corresponding sample type's README file (e.g. using ERS/SRS accession codes for ENA/SRA)
- [x] Does your PR pass validation checks?
- [ ] Changelog is updated?

ivelsko commented:
Analysis of oral microbiome from fossil human remains revealed the significant differences in virulence factors of modern and ancient Tannerella forsythia
https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-06810-9

jfy133 commented:
Link above is for the later 2020 paper.

Original: https://academic.oup.com/gigascience/article/6/7/gix044/3867068

jfy133 commented on ancient-microbiome/ancient-microbiome.tsv:
```diff
@@ -163,4 +163,169 @@ Ottoni2019	2019	10.1038/s41598-019-56074-x	Gabbanat al-Qurud	25.741	32.592	Egypt
+Zhou2018	2018	10.1016/j.cub.2018.05.058	St. Olav's Cathedral	63.425	10.392	Norway	SK152	Homo sapiens	800	NA	oral	dental calculus	NA	UCPH ERDA	E56xgi8CEl
+Philips2017	2017	10.1186/s12864-020-06810-9	Gniezno	52.538	17.597	Poland	PCA0161	Homo sapiens	900	10.1186/s12864-020-06810-9	oral	tooth	NA	SRA	SRS1815425
+Philips2017	2017	10.1186/s12864-020-06810-9	Gniezno	52.538	17.597	Poland	PCA0162	Homo sapiens	900	10.1186/s12864-020-06810-9	oral	tooth	NA	SRA	SRS1815300
+Philips2017	2017	10.1186/s12864-020-06810-9	Kowalewko	52.534	17.582	Poland	PCA0002	Homo sapiens	1800	10.1186/s12864-020-06810-9	oral	tooth	NA	SRA	SRS1815290
```
When I google maps, Kowaleko, I get: 52.699, 17.605, but the one here looks very close to Gniezno. Please verify.

jfy133 commented:
I looked again at the Willman paper #100 , and indeed the bone samples do not show any sign of oral taxa which to me wouldn't be that helpful in this particular list. I would suggest drop the samples tin this Philips paper that show no sign of oral taxa as in fig 4.

ivelsko commented:
I emailed the corresponding author of the study to ask for help clarifying the sample codes and ENA/SRA codes

ivelsko commented:
I added the sample ages from radiocarbon dates for the samples that had them, and fixed the lat/long for Kowaleko. To remove the samples with no oral taxa, what cut-off is appropriate? From Sup. Table S1 the % oral prokaryote is >= 50% in only 13 samples, and >= 20% in only 21 samples. Basically, most of the samples have a dominant environmental signal. 

If we're still including samples that have no damage signal, why would we cut out ones with no oral signal? It makes sense to me to leave these in too, and users are responsible for checking, just like with the damage signals

ivelsko commented:
@maxibor I can't figure out what the issue is with the failed check error message, it's all python errors, can you help me out?

maxibor commented:
@ivelsko The error comes from the line 106 here: https://github.com/SPAAM-community/AncientMetagenomeDir/blob/5921780492c6d4483712a23cd68194d5772b225b/ancientmetagenome-hostassociated/ancientmetagenome-hostassociated_schema.json#L106

It's using the old URL on the philips2017 branch (`spaam-workshop` instead of `spaam-community`)

jfy133 commented:
@ivelsko Ah yeah Maxime is right. It might be worth for both PRs to make a brand new branch (from current Master) and copy and paste in. Given the PR is so old, There might be some bsaic changes to double check too (extra columns etc.) although not many for this list.

jfy133 commented:
> I looked again at the Willman paper #100 , and indeed the bone samples do not show any sign of oral taxa which to me wouldn't be that helpful in this particular list. I would suggest drop the samples tin this Philips paper that show no sign of oral taxa as in fig 4.



I go back on this statement. If it's antonomically correct to look for oral taxa, then keep all samples (even if they don't have any preservation). In the case of Willman, they used bone as a negative control for oral taxa essentially, so I think it's OK to exclude those bone samples, as oral taxa is the main aim of the study.

maxibor commented:
Merged with #354
